qabelbox/src/main/AndroidManifest.xml

```diff
@@ -142,6 +142,7 @@
                 android:name="android.accounts.AccountAuthenticator"
                 android:resource="@xml/authenticator"/>
         </service>
+        <service android:name=".chat.service.AndroidChatService" />
```

Comment:
You can register your intent filter right here instead of the QabelBoxApplication